tslint.json

```diff
@@ -62,7 +62,7 @@
             "check-open-brace",
             "check-whitespace"
         ],
-        "quotemark": [true, "single", "avoid-escape", "jsx-single"],
+        "quotemark": [true, "single", "avoid-escape", "jsx-double"],
```

Comment:
Curious to hear your thoughts on the change to double. Just personal pref? I'm flexible

Reply:
@andrewpottenger this is to support prettier. It does not support flexible settings with typescript/react projects without tricks and workarounds, so this is just to save time for intendations etc